charts/opentelemetry-demo/Chart.lock

```diff
@@ -1,18 +1,18 @@
 dependencies:
 - name: opentelemetry-collector
   repository: https://open-telemetry.github.io/opentelemetry-helm-charts
-  version: 0.109.0
+  version: 0.107.0
```

Comment:
@puckpuck do you want to decrease these versions?

Reply:
Thanks for catching this. Certainly did not intend to decrease these.